context/context.go

```diff
@@ -186,8 +203,15 @@ func (context *AMFContext) AllocateRegistrationArea(ue *AmfUe, anType models.Acc
 }
 
 func (context *AMFContext) NewAMFStatusSubscription(subscriptionData models.SubscriptionData) (subscriptionID string) {
-	// id, err := amfStatusSubscriptionIDGenerator.Allocate()
-	id, err := context.Drsm.AllocateInt32ID()
+	var id int32
+	var err error
+	if context.EnableDbStore {
+		id, err = context.Drsm.AllocateInt32ID()
+	} else {
+		var tmp int64
+		tmp, err = amfStatusSubscriptionIDGenerator.Allocate()
```

Comment:
Similar comment related to types, let's avoid this as much as possible.

Reply:
At the very least, let's check for errors with `ok`.